README.md

```diff
@@ -41,7 +41,7 @@ Besides functionality to perform a typical recommend-measure loop, BayBE's highl
 - 🔄 All objects are fully de-/serializable: Useful for storing results in databases or use in wrappers like APIs
 
 
-## Quick Start
+## ⚡ Quick Start

```

Comment:
Specifically for the further emojification, could you built the doc on your fork and provide a link here?

Reply:
No need to build a fork, you can just go on the main github page and select the branch there. The readme will then show the updated content immediately

Reply:
@AVHopp: if happy, please resolve (or suggest alternatives / vote against)